### PR TITLE
Added Web Developer JD

### PR DESCRIPTION
**Pull request by LoznianuAnamaria**
Added the JD for the WEB developer we are searching for the FrontEnd team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - 🐬 [Senior Blockchain Developer](/sr-dev.md)
 - 🐙 [Application Engineer - Enterprise Team](/ae.md)
 - 🦈 [Senior Blockchain Developer - Data Farming](/df.md)
+- 🐳 [Senior Web Developer](/web.md)
```
Comment (trentmc) [inline README.md]: Datapoints:
- "Senior blockchain developer" was written to be general enough for both FE or BE devs. 
- devs like "blockchain" in the title (or "web3", "smart contract", ..). Especially if blockchain is the major part of their work.

It's ok to have more specialized roles, like the proposal here. Really fine. Then also let's:
- replace "Senior Blockchain Developer" with "Senior Backend Blockchain Developer"
- and change "Senior Web Developer" title to have something blockchain. Ideas: "Senior Frontend Blockchain Developer", "Senior Frontend Web3 Developer",

File: web.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Ocean has a vibrant, expanding ecosystem of data suppliers, data consumers, stakers, and community developers. To serve it better, we’re hiring! If you are someone that loves to build things and cares deeply about the future of data, we want to hear from you.
+
+
+## Responsibilities
+* You will design & build Ocean Protocol core software with your team in an agile, open-source development process
+* You will collaborate closely with product designers with a focus on new feature design and prototyping
+* Be involved in the user experience design process, from wireframes and mockups through implementation and release
+* Help define and maintain visual guidelines and ensure consistency throughout the apps
+* Fast prototyping of new user interactions and iteration based on feedback from peers
```
Comment (trentmc) [inline web.md]: Maybe somewhere here, emphasize that it's _frontend_ work.

File: web.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Skills and Experience 
+* We don’t expect applicants to be an expert in everything listed below which is why we’ve separated the requirements into two tiers. The ideal candidate loves to build user-friendly interfaces, pays attention to improve the aesthetics and the functionality of the site, and is very eager to learn.
+
+## Minimum Qualifications
+* 5+ years of software experience
+* 2+ years Javascript experience
+* 6+ months web3.js experience
+* 1+ years blockchain frontend (dapp) 
+* Proficient with Figma app
```
Comment (trentmc) [inline web.md]: Looks fine to me. I'm not as deep in FE but it passes my sanity check

**Review by trentmc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LoznianuAnamaria) [reply README.md]: In my perspective, the Senior Blockchain Developer job description is more like a FullStack engineer and not just backend. 
I added this specific Frontend Job description for the person we are searching for to build UI on our market, but blockchain(web3) experience is still a requirement

Comment (trentmc) [reply README.md]: Yes, that's fine. But you need a title more Web3-y than "Senior Web Developer", that's all:)

**Review by mihaisc — APPROVED** [review]

**Pull request merged**